### DISCUSSION TITLE
Set Scheme without Power On

### DISCUSSION THREAD

**Discussion by githubwesseln** [arendst/Tasmota#14066] [Ideas]
With "SetOption20 = 1" I can prevent an automatic "Power On" for setting "Color", I need this for setting "Scheme".

**Comment by sfromis** [comment]
As mentioned on Discord, the [code](https://github.com/arendst/Tasmota/blob/development/tasmota/xdrv_04_light.ino#L2684-L2723) for the Scheme command consistently sets power on, irrespective of the SetOption.

**Reply by githubwesseln** [reply]
Ok, I will change the code and test it, thanks for this info.
